### DISCUSSION TITLE
Duplicate_sensors.py - HELP

### DISCUSSION THREAD

**Discussion by EndPoint-dev-Brazil** [CrowdStrike/falconpy#1150] [Ideas]
Would it be possible to put just two validations in this script, could you help me edit it to compare just the "Hostname / IP"
Link Reference: https://github.com/CrowdStrike/falconpy/blob/main/samples/hosts/duplicate_sensors.py

**Comment by jshcodes** [comment]
Hi @EndPoint-dev-Brazil - 

Providing a way to adjust the comparison should be doable.  

We will post a reply for you here soon. 😄

**Reply by EndPoint-dev-Brazil** [reply]
Tanks
